SecretSanta/src/SecretSanta.Business/GroupRepository.cs

```diff
@@ -45,5 +73,19 @@ public void Save(Group item)
 
             MockData.Groups[item.Id] = item;
         }
+
+       //Link to the location of a shuffle method that I found: https://stackoverflow.com/questions/273313/randomize-a-listt
+       private void Shuffle<T>(this IList<T> list)  
```

Comment:
This is declared like an [extension method](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods), however extension methods must live inside of static classes (hence the current compilation error). However, since you are just using it like a normal method you can simply drop the `this` in front of the parameter.
```suggestion
       private void Shuffle<T>(IList<T> list)  
```